app/controllers/hyrax/controller.rb

```diff
@@ -4,6 +4,7 @@ module Hyrax::Controller
   included do
     class_attribute :create_work_presenter_class
     self.create_work_presenter_class = Hyrax::SelectTypeListPresenter
+    self.search_state_class = Hyrax::SearchState
```

Comment:
I suspect that the whole controller.rb file is not necessary and could be deleted now. Could you give it a try?